physics/module_mp_thompson.F90

```diff
@@ -2006,6 +2021,7 @@ subroutine mp_thompson (qv1d, qc1d, qi1d, qr1d, qs1d, qg1d, ni1d,    &
       odt = 1./dt
       odts = 1./dtsave
       iexfrq = 1
+      av_i = av_s * D0s ** (bv_s - bv_i)
```

Comment:
Does this need a comment to explain the expression? It's not clear to me as a non-microphysics expert where the formula comes from since this was changed from a constant.

Reply:
Added one line of comment as suggested.